src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,154 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private static final String INDEX_EXCEPTION = "Index is incorrect";
+    private Node<T> node;
```

Comment:
why do you need this variable? Don't you think it's redundant? You already have a tail, so you can just use local variables in method instead of making it a class variable